core/market/decentralized/src/db/model/agreement.rs

```diff
@@ -111,6 +113,53 @@ impl Agreement {
             committed_signature: None,
         }
     }
+
+    pub fn into_client(self) -> Result<ClientAgreement, ErrorMessage> {
+        let demand_properties = serde_json::from_str(&self.demand_properties)
+            .map_err(|e| format!("Can't serialize Demand properties. Error: {}", e))?;
+        let offer_properties = serde_json::from_str(&self.offer_properties)
+            .map_err(|e| format!("Can't serialize Offer properties. Error: {}", e))?;
+
+        let demand = Demand {
+            properties: demand_properties,
+            constraints: self.demand_constraints,
+            demand_id: Some(self.demand_id.to_string()),
```

Comment:
Misleading: properties aren't related to demand, but to last proposals

Reply:
but this comment relates more to `ya-client`